PubNub/Core/PubNub+Subscribe.m

```diff
@@ -83,7 +83,7 @@ - (void)subscribeToChannels:(NSArray *)channels withPresence:(BOOL)shouldObserve
         presenceChannelsList = [PNChannel presenceChannelsFrom:channels];
     }
     [self.subscriberManager addChannels:[channels arrayByAddingObjectsFromArray:presenceChannelsList]];
-    [self.subscriberManager subscribe:YES usingTimeToken:timeToken withState:state completion:nil];
+    [self.subscriberManager subscribe:YES usingTimeToken:timeToken withState:state region:nil completion:nil];
```

Comment:
@jzucker2 I think time treated parameters should be declared together. Please switch _withState:_ and _region:_ places.